ArraySorting.cls

```diff
@@ -0,0 +1,53 @@
+/*3. Method which will accept array of integers as parameters write a code to
+a. Sort the array in ascending order
+b. Sort the array in descending order
+c. Print the Maximum number in the array
+d. Print the Minimum number in the array
+e. Print duplicates and its occurrences in the array*/
+
+//a. Sort the array in ascending order
+public class ArraySorting{
+	public static void arrayOrder(){
+		Integer[] arrayList = new List<Integer> {2,2,3,3,3,4};
+		
+		//Sort the array in ascending order
+		arrayList.sort();
+		System.debug('Ascending Order: '+ arrayList);
+		
+		//Sort the array in descending order
+		Integer[] descList = new List<Integer>();
+		for(Integer i = arrayList.size() - 1 ; i >= 0; i--){
+			descList.add(arrayList.get(i));
+		}
+		System.debug('Descending Order: '+ descList);
+		
+		// Print the Maximum number in the array
+		Integer maxNo = descList[0];
+		for(Integer temp = 0 ; temp < descList.size() ; temp++){

```

Comment:
we require loop to traverse the array.. If we don't use loop we have to implement a recursive function to traverse through the array